ts/features/itwallet/common/utils/itwTypesUtils.ts

```diff
@@ -46,4 +61,5 @@ export type StoredCredential = {
   parsedCredential: ParsedCredential;
   credentialType: string;
   issuerConf: IssuerConfiguration;
+  storedStatusAttestation?: StoredStatusAttestation;
```

Comment:
As a rule of thumb, changing a persisted type always requires a migration, even if it might be undefined like this one.

Reply:
I added a noop migration for consistency in bbf9364df37137389b5aeb9bab3658652ae70bc8, `storedStatusAttestation` was added with backward-compatibility in mind.

Reply:
Personally I would destructure the object and set it to `undefined` just to be explicit as we might need to perform another migration in the future. Does it make sense to you?

Reply:
Addressed in 9d1951034a7f2a2880400a6361267c697ddbe003.